### ISSUE TITLE
About the segmentation.

### ISSUE THREAD

**Issue by wangshihuang** [pytorch/vision#2832]
In the reference, I replaced the cross entropy in the semantic segmentation module with weighted cross entropy. The result was worse. The weight is calculated based on the training set. If the cross entropy is replaced by focal loss, the effect is also poor. Why is this? Still, the best loss function for semantic segmentation is cross entropy.
I sincerely need your help!

**Comment by wangshihuang** [comment]
How should i analyze.

**Comment by vfdev-5** [comment]
@wangshihuang please, use https://discuss.pytorch.org/ forum which is more appropriate for questions like that.

**Comment by vfdev-5** [comment]
Otherwise, there is nothing "wrong" to use cross entropy as loss. 

> I replaced the cross entropy in the semantic segmentation module with weighted cross entropy. The result was worse

Check per class performances, check the computed weights.

> How should i analyze.

There are various resources online to help with how to analyze the training. For example
http://karpathy.github.io/2019/04/25/recipe/
